### ISSUE TITLE
New rule: "import-rsvp/jquery"

### ISSUE THREAD

**Issue by Turbo87** [ember-cli/eslint-plugin-ember#61]
Instead of accessing `Ember.$` and `Ember.RSVP` we should:

```js
import $ from 'jquery';
import RSVP from 'rsvp';
```

I think it might be a good idea to encode that in an ESLint rule

**Comment by michalsnik** [comment]
Hmm, okay. This makes sense 👍 

But, currently lots of us do smth like:
```
import Ember from 'ember';
const { $, RSVP } = Ember;
```

So we already use `$` and `RSVP` directly which complicates this rule a bit.

I'm thinking about actual implementation of this rule. And it looks like apart from looking for `Ember.$` and `Ember.RSVP` we need to check for the existence of `import $ from 'jquery';` and `import RSVP from 'rsvp'` and then if they don't exist but `$` or `RSVP` are being used - we can assume they're from `Ember` namespace so we can point that they are supposed to be imported directly, not destructed from Ember.

What do you think @Turbo87 about naming this rule `require-direct-imports` or something similar? We can mention in documentation which modules are now supported. In the future I guess there will be more of them, so making it more generic now might have a sense.

**Comment by Turbo87** [comment]
FWIW https://github.com/ember-cli/ember-rfc176-data has more information on the suggested new import paths

**Comment by michalsnik** [comment]
I'm wondering if we should add this rule or create one that will report all global imports regarding this list: https://github.com/ember-cli/ember-rfc176-data/blob/master/globals.json
